### ISSUE TITLE
Error deleting ConfigConnectorContext

### ISSUE THREAD

**Issue by maqiuyujoyce** [GoogleCloudPlatform/k8s-config-connector#523]
Config Connector user of v1.50.0 or above may run into the following error (or a similar one) when deleting the `ConfigConnectorContext`:

```
  status:
    errors:
    - 'error during reconciliation: error verifying the Config Connector resource
      counts in namespace ''example-1234-321423'': error listing objects for gvk ''core.cnrm.cloud.google.com/v1alpha1,
      Kind=ServiceMapping'': servicemappings.core.cnrm.cloud.google.com is forbidden:
      User "system:serviceaccount:configconnector-operator-system:configconnector-operator"
      cannot list resource "servicemappings" in API group "core.cnrm.cloud.google.com"
      in the namespace "example-1234-321423"'
    healthy: false
```

**Comment by maqiuyujoyce** [comment]
Since [v1.50.0](https://github.com/GoogleCloudPlatform/k8s-config-connector/releases/tag/v1.50.0), `ServiceMapping` CRD has been removed from Config Connector installation bundle. If you upgraded from a version below v1.50.0 to v1.50.0 or above, the `ServiceMapping` CRD and its CRs may still exist, but can no longer be managed/handled by Config Connector, which causes the error. We are working on a fix and will update this issue when we have any new information.

For now, you can work around it by deleting all the `ServiceMapping` CRs and the `ServiceMapping` CRD manually before deleting the `ConfigConnectorContext`.

**Comment by maqiuyujoyce** [comment]
Hi @travisrandolph-bestbuy , the workaround will be released in Config Connector 1.61.0.

**Comment by maqiuyujoyce** [comment]
Hi @travisrandolph-bestbuy , please ignore my last comment. The fix has been released in Config Connector 1.60.0. Please let us know if you have any questions.
